diff --git a/source/encoder/parameters.c b/source/encoder/parameters.c
index 9adec5b..3d04a7c 100644
--- a/source/encoder/parameters.c
+++ b/source/encoder/parameters.c
@@ -528,7 +528,7 @@ char *copy_parameter(char *dst, const char *src)
  *          : less than 2M bytes
  * ---------------------------------------------------------------------------
  */
-static char *xavs2_get_configs(int argc, const char * const *argv)
+static char *xavs2_get_configs(int argc, char **argv)
 {
     const int size_file_max = 1 << 20; // 1MB
     char item[4096];
